roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -23,7 +23,7 @@
   when: openshift.common.is_containerized | bool and not openshift.common.is_master_system_container | bool
 
 # workaround for missing systemd unit files
-- name: Create the systemd unit files
+- name: "Create the {{ openshift.common.service_type }} systemd unit file"
```

Comment:
I feel like vars in task names is relatively new. I noticed it working recently too.